Description: Prevent unsafe parallel building of manpages with autogen
Author: Mike Miller <mtmiller@debian.org>
Bug: https://gitlab.com/ocserv/ocserv/issues/76
Last-Update: 2016-12-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -20,3 +20,4 @@ ocpasswd.8: ../src/ocpasswd/args.def
 	@AUTOGEN@ -L../src -DMAN_SECTION=8 -Tagman-cmd.tpl "$<".tmp && \
 	rm -f "$<".tmp
 
+.NOTPARALLEL:
